db/migrate/20230126135721_convert_address_to_uk_address.rb

```diff
@@ -0,0 +1,9 @@
+class ConvertAddressToUkAddress < ActiveRecord::Migration[7.0]
+  def up
+    Page.where(answer_type: "address", answer_settings: nil).update_all(answer_settings: { input_type: { uk_address: "true", international_address: "false" } })
+  end
+
+  def down
+    Page.where(answer_type: "address", answer_settings: { input_type: { uk_address: "true", international_address: "false" } }).update_all(answer_settings: nil)
```

Comment:
As with the dates - maybe this should cover all of our cases:
```suggestion
    Page.where(answer_type: "address", answer_settings: { input_type: { uk_address: "true", international_address: "false" } }).update_all(answer_settings: nil)
    Page.where(answer_type: "address", answer_settings: { input_type: { uk_address: "false", international_address: "true" } }).update_all(answer_settings: nil)
    Page.where(answer_type: "address", answer_settings: { input_type: { uk_address: "true", international_address: "true" } }).update_all(answer_settings: nil)
```

Reply:
Same as dates
```suggestion
    Page.where(answer_type: "address").update_all(answer_settings: nil)
```